spec/maxitest_spec.rb

```diff
@@ -61,6 +61,13 @@
     result.should include "3 runs, 1 assertions, 0 failures, 0 errors, 2 skips"
   end
 
+  it "does not interfere with Pride plugin" do
+    result = sh("ruby spec/cases/proud.rb")
+    result.gsub!(/\d+\.\d+/, 'X')
+    result.should include "\e[38;5;154mF\e[0m\e[38;5;154ma\e[0m\e[38;5;148mb\e[0m\e[38;5;184mu\e[0m\e[38;5;184ml\e[0m\e[38;5;214mo\e[0m\e[38;5;214mu\e[0m\e[38;5;208ms\e[0m\e[38;5;208m \e[0m\e[38;5;203mr\e[0m\e[38;5;203mu\e[0m\e[38;5;198mn\e[0m in Xs, X runs/s, X assertions/s."
```

Comment:
Bear with me ... the brittle part is the long ANSI-riddled string, right? The assertion on line 68 is sufficient to demonstrate the failure. I can just delete lines 66-67 as long as that would be clear enough for anyone reading the test.